front/src/applications/editor/components/LinearMetadata/FormComponent.tsx

```diff
@@ -41,7 +41,9 @@ export type FormContext = {
   isCreation: boolean;
 };
 
-const IntervalEditorComponent = (props: FieldProps<GeoJsonProperties, RJSFSchema, FormContext>) => {
```

Comment:
Sounds like the type I've written [here](https://github.com/OpenRailAssociation/osrd/pull/8884/files#diff-66b0673a768e8de8c885355417a57d1e5b21e151f544e934741984f87a9f589dR44) was incorrect, but I missed this because `GeoJsonProperties` is a record of `any` values so no compiler error was generated by `formData?.filter(notEmpty)` below.